windows-installer/fluent-template.conf

```diff
@@ -1,5 +1,7 @@
 # Master configuration template file for google-fluentd on windows.
 
+@include CUSTOM_CONFIG_DIR_PLACE_HOLDER
```

Comment:
Nit: technically, this is not a custom config **dir**(ectory), because it ends up being a glob. How about just `CUSTOM_CONFIG_PLACE_HOLDER`?